src/pynwb/ndx_patterned_ogen/patterned_ogen.py

```diff
@@ -480,27 +525,72 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         for key, val in args_to_set.items():
             setattr(self, key, val)
+        columns = popargs("columns", kwargs)
+        if columns is not None:
+            colset = {c.name: c for c in columns}
+            # First check: power_per_rois and power must not be defined in the same time
+            if "power_per_rois" in colset.keys() and "power" in colset.keys():
+                raise ValueError(
+                    "Both 'power' and 'power_per_rois' have been defined. Only one of them must be defined"
+                )
```

Comment:
Shouldn't there be analogous checks like this for `frequency` and `pulse_width` too?

Reply:
Right! Fixing this now.